web/src/components/Prices/_HollaexToken.scss

```diff
@@ -0,0 +1,254 @@
+.hollaex-token-wrapper {
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+
+	.wizard-container .ant-input-affix-wrapper,
```

Comment:
`wizard-container` seems off here. Also, we have some classes defined for the ant select, see if we can reuse them.